docs/k8s/getting-started-kic.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Getting Started - Kubernetes Ingress Controller
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import K8sIngressInstallExample from "/examples/k8s/install-ingress.mdx";
+import K8sGatewayAPIInstallExample from "/examples/k8s/install-gateway-api.mdx";
+
+import K8sIngressModifyHeadersExample from "/examples/k8s/http-request-headers.mdx";
+import K8sGatewayModifyHeadersExample from "/examples/k8s/http-request-headers-gateway.mdx";
+
+import K8sIngressRedirectExample from "/examples/k8s/http-request-redirect.mdx";
+import K8sGatewayRedirectExample from "/examples/k8s/http-request-redirect-gateway.mdx";
+
+import StaticDomainKubernetesExample from "/examples/k8s/http-static-domain.mdx";
+import StaticDomainKubernetesGatewayExample from "/examples/k8s/http-static-domain-gateway.mdx";
+
+import StaticDomainKubernetesPathExample from "/examples/k8s/http-static-domain-path.mdx";
+import StaticDomainKubernetesGatewayPathExample from "/examples/k8s/http-static-domain-gateway-path.mdx";
+
+# Kubernetes Ingress Controller
+
+## Overview
+
+The Kubernetes Ingress Controller within the Kubernetes Operator allows access to k8s Services using Kubernetes [Ingress CRDs](https://kubernetes.io/docs/concepts/services-networking/ingress/). More details on how these are derived is described in the [guide to edges in k8s](/docs/k8s/with-edges).
+
+Other ngrok features such as TCP Edges can be configured via [CRDs](/docs/k8s/user-guide/crds).
+
+If you are looking to install the controller for the first time, see our [deployment-guide](/docs/k8s/deployment-guide/).
+If it's already installed and you are looking to configure ingress for an app or service, see our [user-guide](/docs/k8s/user-guide/).
+
+## Install
+
+Installing the ngrok Operator is easy using Helm.
+
+<K8sIngressInstallExample />
+
+<br />
+
+## Create an edge with a domain
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic to the Kubernetes service `example-service` running on port 80.
+
+<StaticDomainKubernetesExample />
+
+## Route Traffic based on a path
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from example.ngrok.app/foo to the Kubernetes service `example-service` running on port 80.
```

Comment:
```suggestion
This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from `example.ngrok.app/foo` to the Kubernetes service `example-service` running on port 80.
```